cirq-core/cirq/qis/clifford_tableau.py

```diff
@@ -509,11 +509,14 @@ def destabilizers(self) -> List['cirq.DensePauliString']:
         generators above generate the full Pauli group on n qubits."""
         return [self._row_to_dense_pauli(i) for i in range(self.n)]
 
-    def _measure(self, q, prng: np.random.RandomState = np.random) -> int:
+    def _measure(self, q, prng: Optional[np.random.RandomState] = None) -> int:
         """Performs a projective measurement on the q'th qubit.
 
         Returns: the result (0 or 1) of the measurement.
         """
+        real_prng: np.random.RandomState = (
+            random_state.parse_random_state(np.random) if prng is None else prng
```

Comment:
`parse_random_state` already returns np.random for None.
```suggestion
            random_state.parse_random_state(prng)
```